### PR TITLE
Refactor requirement check in install

### PR DESCRIPTION
**Pull request by aledeg**
Changes proposed in this pull request:

- Add requirement check template during install

How to test the feature manually:

1. Make a new FreshRSS install

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

I've extracted the template in the check requirement to
ensure that future modifications will be easier.

### CODE REVIEW

File: app/install.php [hunk 1]
```diff
@@ -362,108 +362,39 @@ function printStep0() {
 <?php
 }
 
+function printStep1Template($key, $value, $messageParams = []) {
+	$message = _t("install.check.{$key}.{$value}", ...$messageParams);
```
Comment (Alkarex) [inline app/install.php:366]: Could we generalise the translation to have only one string instead of many?

Comment (aledeg) [reply app/install.php:366]: We could but I am not sure it will work. I have to think about it.

Comment (Alkarex) [reply app/install.php:366]: While it would be nice to remove the many translation strings we have for that, it can also wait for later

Comment (aledeg) [reply app/install.php:366]: Actually, I think the whole install process needs a bit of love. I'll look into it when I have time and feel like it.

Comment (Alkarex) [reply app/install.php:366]: Should we then merge this PR as is while waiting for future improvements?

Comment (aledeg) [reply app/install.php:366]: I think it's better if you do not want to see this one collect dust

**Comment by aledeg** [comment]
I've messed up the translation in that one. I fixed it in #3373

File: app/install.php [hunk 2]
```diff
@@ -362,108 +362,39 @@ function printStep0() {
...
-
-	<?php if ($res['http_referer'] == 'ok') { ?>
-	<p class="alert alert-success"><span class="alert-head"><?= _t('gen.short.ok') ?></span> <?= _t('install.check.http_referer.ok') ?></p>
-	<?php } else { ?>
-	<p class="alert alert-error"><span class="alert-head"><?= _t('gen.short.damn') ?></span> <?= _t('install.check.http_referer.nok') ?></p>
-	<?php } ?>
+	<?php
+	printStep1Template('php', $res['php'], [PHP_VERSION, FRESHRSS_MIN_PHP_VERSION]);
+	printStep1Template('pdo', $res['pdo']);
+	printStep1Template('curl', $res['curl'], [$version['version']]);
```
Comment (Alkarex) [inline app/install.php:383]: @aledeg Ups, I did not spot it, but the variable `$version` is undefined

Comment (aledeg) [reply app/install.php:383]: Good catch, fixed in #3427

**Pull request merged**